### ISSUE TITLE
How do I delete my Codidact account? 

### ISSUE THREAD

**Issue by ShashankVM** [codidact/qpixel#470]
I am user Anon (https://meta.codidact.com/users/53813) on Codidact Meta and other Codidact sites. I wish to delete my account from Codidact, I logged into my profile but could not find the delete button. How do I delete my profile?

**Comment by luap42** [comment]
Account has been deleted, per request.

**Comment by ShashankVM** [comment]
According to [Codidact's goal](https://github.com/codidact/docs)
> We also believe that Q&A communities should be free from the politics and shenanigans of private, profit-focused companies.

I believe Q&A communities should be free from the politics and shenanigans.

Thank you @luap42  for deleting my account.

**Comment by FoggyFinder** [comment]
See also [How can one remove their account ?](https://meta.codidact.com/posts/277522)

> I wish to delete my account from Codidact

May I ask why?

**Comment by ShashankVM** [comment]
> See also [How can one remove their account ?](https://meta.codidact.com/posts/277522)
> 
> > I wish to delete my account from Codidact
> 
> May I ask why?

I assume this question is directed to me, so here is my answer:

@FoggyFinder this is the reason https://github.com/codidact/docs/issues/33 
In case they delete that too, here it is: 
![Double standards for development team and user community due to the site's design · Issue #33 · codidact_docs](https://user-images.githubusercontent.com/58113556/113404848-74f5b700-93c6-11eb-9780-8677d30a0cd1.png)

**Comment by ArtOfCode-** [comment]
@ShashankVM Just a small clarification. I found the issue you posted on the docs repository a little confusing, as it says there's a bug but no bug is identified in the description - there's a discussion of how the community is run and in particular you raise an issue with this particular feature, but I can't see any bugs identified.

As for the response to your question on Meta - I'm sorry you were made to feel disrespected; that's not the environment we're trying to create. I've just now gone through that page in full. As far as I can see, all responses you received from Codidact staff were in line with our Code of Conduct. There are a small number of comments from community members that seem confrontational on a first read; the flag function exists for instances like this, which I presume you used to being those comments to moderators' attention, and those reports will be reviewed shortly.

My emails are always open at art@codidact.org if you have other feedback you'd like to pass on. You're always welcome back any time you'd like to.

**Comment by ShashankVM** [comment]
@ArtOfCode- thank you for your comment. The bug is in the *design* of the platform, not in the code or **implementation**. If the software design is wrong, how can the implementation be right? As I have noted in my issue, the bug is that the developers have an unfair advantage over the regular community. Developers are inserting "easter eggs" into the platform **without informing the user community**, which might turn out to be a **nasty surprise** for users who control their mouse using a keyboard. Can you now see the bug? Also, I have reason to believe that the "easter egg" which was inserted created a vulnerability to the website.  This "easter egg" is not different from **malware** given that malware is software which the end user has no idea about and malware is inserted with malicious intent. Also the developer team is getting free rep by posting ominous messages on Codidact Meta, such as "A note on April 1st" - which is sure to scare away users who suffer from paranoia.

**Comment by ShashankVM** [comment]
@ArtOfCode-  I prefer a public and open discussion over a private email for such matters which are not of importance to me. By keeping the conversation public, other users can also chime in. Again, converting my deleted account to "system user" does not seem like the right thing to do, considering future deleted accounts and posts by other users would also get owned by 'system users'. I have also realised such Q&A platforms which are excessively focused on gamification are not productive compared to other forums which are more interested in solving problems.

**Comment by ShashankVM** [comment]
@ArtOfCode-  Moderation on Codidact is an "ability" not a responsibility. With great power comes great responsibility. If users with Moderation "super-power" think they can delete any post *they want to* then moderation becomes a joke, that is on the moderators, who become clowns, not respected users who are known for taking the right decisions and *doing the right thing*. Respect is earned, and I can say neither the moderation team of Stack Exchange nor of Codidact has earned my respect since they are too biased and too unfair. Since you are on Stack Exchange, you might be interested in looking at this too: https://github.com/ShashankVM/meta_stack

The problem on Stack Exchange and Codidact is the same thing: Abuse of power by **both the moderation team and the people behind the site**. Community users who bring value and money to these sites are treated like a doormat.

**Comment by ShashankVM** [comment]
@ArtOfCode-   silently closing my account and the issue I created without **solving the problem** by *creating a delete button* seems very shady, and I think the team behind Codidact has some vested interests. Codidact's readme.md file (https://github.com/codidact/docs) points to top answers project which seems to compete with stack overflow. So codidact and top answers project seem too intertwined to be considered entirely separate projects. Creating a Codidact open source project seems like a **plot to lure developers to work for free** on the software that the profit making company (ironically claiming to be not-for-profit, but that's debatable) behind this whole facade wants to use to make money. It is like the "open source" android project that Google created to lure developers to work for free. We now know who owns Android.

**Comment by ArtOfCode-** [comment]
@ShashankVM I'm sorry you feel that way. Thank you for clarifying your bug report - a "bug" in how the software is owned and run is not a bug, just a discussion or complaint. As I said, you're airways welcome back.

**Comment by ShashankVM** [comment]
@ArtOfCode-  We can go on and on about discussing whether it should be called a bug or not. I say it is a bug in the Q&A Platform, you say it is not. A bug in the algorithm or software design *is still a bug* that can be fixed. You might even want to call the "easter egg" a feature if you want to. We'll have to agree to disagree here, since it is my conviction that Q&A should be free from shenanigans and politics, in order to have a productive discussion/forum. This is clearly is not true for Codidact. For your information, the "easter egg" was rolled out on all Codidact sites. I checked on Electrical Codidact too.

I have more respect for Stack Exchange Inc. since they wrote an official apology letter to the moderators they unjustly fired and to the community. Stack Exchange Inc. shows they care for the community, and they have to do that, since the community brings them traffic and money. Codidact's team showed an indifferent attitude and failed to recognise or acknowledge the problem in the system, that was pointed out by me. They are not apologising nor are they promising to not deliver such malware in the future. I don't expect an apology and I am not going to use Codidact again, but I hope the Codidact team would have the decency to acknowledge that such "easter eggs" can be disastrous to users with mobility impairments who use the keyboard to control the mouse, and promise to not allow such "easter eggs" in the future. 

I don't need to use Codidact nor Stack Overflow, nor do I wish to do the thankless job of getting traffic to these Q&A platforms by answering questions. 

I was a former contributor in the OpenHW Group, then I decided to not contribute since I realised that while the organization was not-for-profit they still make money through memberships, yet they did not want to pay me for contributing, so I dropped the task given, did not turn in my work and left the organization. I also realised the OpenHW group project was not completely open-source. Since then, I am wary of not-for-profit organizations.

On the other hand, there are also good projects like KDE and Qt. I am a huge fan of KDE and Qt and regularly use their software.
